src/start_server.ts

```diff
@@ -151,6 +156,32 @@ export function getApp(options: ServerOptions): express.Express {
 
   app.use('/components/', polyserve);
 
+  if (options.proxy) {
+    if (options.proxy.path === 'components') {
+      console.error('proxy path can not start with components.');
+      return;
+    }
+
+    const proxyPath = options.proxy.path
+                             .replace('*', '\*')
```

Comment:
There's two problems here. 1) the first argument to replace() will match even escaped '*'s 2) the right hand side needs to escape the backslash.

I think this is actually easier to write without regexes. This will probably work, or maybe there's a package that handles this:

```typescript
let escapedPath = '';
let escaped = false;
for (const char of options.proxy.path) {
  escaped = (char === '\\') ? !escaped : false;
  if (escaped && (char === '*' || char === '+' || char === '?')) {
    escapedPath += '\\';
  }
  escapedPath += char;
}
```